services/scan/src/interpreter.ts

```diff
@@ -241,21 +233,20 @@ export class Interpreter {
       return { interpretation: { type: 'BlankPage' } };
     }
 
-    const result = await getBallotImageData(
-      ballotImageFile,
-      ballotImagePath,
-      detectQrcodeResult
-    );
-
-    if (result.isErr()) {
-      timer.end();
-      return { interpretation: result.err() };
+    if (!detectQrcodeResult.qrcode) {
+      return {
+        interpretation: {
+          type: 'UnreadablePage',
+          reason: 'No QR code found',
+        },
+      };
```

Comment:
Did we previously handle this case elsewhere? If so, can we remove the check there?

Reply:
Good catch, fixed in 919d1bb.